torch/csrc/jit/codegen/cuda/ops/alias.h

```diff
@@ -39,6 +39,24 @@ TORCH_CUDA_CU_API TensorView* squeeze(
 
 TORCH_CUDA_CU_API TensorView* unsqueeze(TensorView* x, int dim);
 
+//! Permute a tensor as specified by axis mappings.
+//!
+//! The transposition mapping is specified with a list of pairs from
+//! new to old positions. Positions are relative to the noReduction
+//! domain.
+//!
+//! \param inp Tensor to transpose
+//! \param new2old vector mapping from new to old positions.
+TORCH_CUDA_CU_API TensorView* permute(
+    TensorView* x,
+    const std::vector<int64_t>& new2old);
```

Comment:
We typically use `int` for specifying axes, e.g., `reductionOp`. I guess the eager mode uses `int64_t`? Either type seems fine, but we should make it consistent.

Reply:
I'll make `transposeOp` handle `int64_t`.
In parser.cpp, we treat `IntList` as `std::vector<int64_t>` while `Int` scalars returned as `int`.